Documentation/installation/index.rst

```diff
@@ -6,39 +6,64 @@
 
 .. _gs_guide:
 
-Getting Started Guides
-======================
+Advanced Installation Guides
+============================
```

Comment:
Probably `gs_guide` should point to the actual getting started guides, whereas `gs_install` should point to the installation guides?

Reply:
Hi, it seems the [Update Networking, Contributor Guides and Reference sections](https://github.com/cilium/cilium/pull/20642/commits/5f06e740f7a155fc00f837bd0b94d8cf98c723ca) commit is missing.

Reply:
Right, there's even a few bits here and there that needed to be adjusted. I added a new commit (on top of those from the feature branch), would you mind taking a look please?